README.md

```diff
@@ -82,7 +82,7 @@ Add the FH Android `aar` library in your `build.gradle` project
 
 ```
 dependencies {
-	compile 'com.feedhenry:fh-android-sdk:2.3.1'
+	compile 'com.feedhenry:fh-android-sdk:2.4.0'
```

Comment:
@wei-lee In general example code and documentation must be updated for each release and regression tested.  If we want we can have templates to generate updated README's in a code quality project.  It is part of our long term goals to automate as much of this as we can.